csrc/executor.cpp

```diff
@@ -1342,34 +1342,25 @@ std::vector<FusionExecutor::GlobalBufferInfo> FusionExecutor::
   return global_buffers;
 }
 
-std::vector<at::Tensor> FusionExecutor::allocOutputSpace(
-    const at::ArrayRef<c10::IValue>& inputs) {
-  auto kernel_inputs = KernelArgumentHolder::createKernelArgumentHolder(inputs);
-  auto expr_eval =
-      executor_utils::bindInputs(kernel_inputs, lowered_->kernel());
-
-  auto output_info =
-      getOutputBufferInfo(kernel_inputs, expr_eval, kernel()->indexType());
-
-  return allocateOutputs(kernel(), output_info, options_.device, expr_eval);
-}
-
-std::vector<FusionExecutor::GlobalBufferInfo> FusionExecutor::
-    getOutputBufferInfo(
-        const KernelArgumentHolder& args,
-        ExpressionEvaluator& expr_eval,
-        DataType index_dtype) {
+//! Return information necessary for allocating output tensors. Input
+//! and output tensors are allowed to alias each other, which is
+//! specified by the list of int pairs of input and output indices
+std::vector<FusionExecutor::GlobalBufferInfo> getOutputBufferInfo(
```

Comment:
Should this be in an anonymous namespace? Looks like it's dropped from the header file.

Reply:
correct!